Wikipedia/Code/WMFDatabaseHouseKeeper.swift

```diff
@@ -33,7 +33,7 @@ import Foundation
  
         */
         
-        guard let thirtyDaysAgoMidnightUTC = daysBeforeDateInUTC(days: -30, date: Date()) else {
+        guard let thirtyDaysAgoMidnightUTC = daysBeforeDateInUTC(days: 0 - WMFExploreFeedMaximumNumberOfDays, date: Date()) else {
```

Comment:
Updated & fixed some logic issues (the helper function wasn't using 'days' and also had an unnecessary UTC conversion)